modules/Mail/Spamgourmet/UserStore.pm

```diff
@@ -0,0 +1,79 @@
+package Mail::Spamgourmet::UserStore;
+use strict;
+use utf8;
+use lib "/path/to/modules";
+use DBD::mysql;
+
+
+# the current user handling code is in the Session.pm - this is just a
+# dummy to create a test user.
+my $_config = 0;
+
+sub new {
+  my $proto = shift;
+  my $class = ref($proto) || $proto;
+  my $self = {};
+  bless $self,$class;
+  my %params = @_;
+  if ($params{'config'}) {
+    $self->{'config'} = $params{'config'};
+  } elsif ($_config) {
+    $self->{'config'} = $_config;
+  }
+  return $self
+}
+
+sub setup_test_user {
+    my $self = shift; 
+    my $now = time();
+    my $activeLC="EN";
+
+    # first set up the user
+   
+    my $sql = "INSERT INTO Users (UserName,RealEmail,Password,TimeAdded,IPAddress,LanguageCode,SessionToken,LastCommand)
```

Comment:
Will there need to be any clean up? Or will it be assumed a new (in-memory?) DB is provisioned each time?

Reply:
Currently we are provisioning a new database in each case because we build in docker. Once this is merged we should optimize and change over to INSERT if not exists